app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -200,13 +196,12 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                         //SharePreference to store image
                         shre = PreferenceManager.getDefaultSharedPreferences(this);
                         SharedPreferences.Editor edit = shre.edit();
-                        edit.putString("image_data", encodedImage);
+                        edit.putString(Constant.IMAGE_DATA, encodedImage);
```

Comment:
@the-dagger, So that I have made Constant.java a helper to manage Key values.
It's not in string.xml